CONTRIBUTING.md

```diff
@@ -1,3 +1,52 @@
+## Set Up a Development Environment
+
+1. `$ git clone git@github.com:ruby/debug.git`
+2. `$ bundle install`
+3. `$ rake` - this will
+    - Compile the C extension locally (which can also be done solely with `rake compile`).
+    - Run tests.
+    - Re-generate `README.md`.
+
+If you spot any problem, please open an issue.
+
+## Run Tests
+
+### Run all tests
+
+```bash
+$ rake test
+# or
+$ ruby bin/test-unit.rb
+```
+
+### Run specific test(s)
+
+
+```bash
```

Comment:
I'm not familiar with test/unit, but `ruby test/some_test.rb` doesn't work?

Reply:
wow actually it works. I'll update this section. 
@ono-max since running the test file directly works (it also works with options), do we still need `bin/test-unit.rb?

Reply:
We don't need it.
I didn't know that. Thanks, @ko1 !

Reply:
I'll remove it then 🙂